package.json

```
@@ -50,7 +50,8 @@
 		"trailingComma": "all",
 		"semi": true,
 		"singleQuote": true,
-		"useTabs": true
+		"useTabs": false,
```

not a personal preference - without these settings, I was seeing unintentional formatting changes (spaces to tabs, 4 wide)

---

This change was made on purpose, as the new industry standard is to use tabs to facilitate accessibility for our visually-disabled colleagues. While I realise that complicates reviews, it is a good change worth the reviewer's effort.

```suggestion
		"useTabs": true,
```